src/virtual_secondary/managedsecondary.cc

```diff
@@ -72,6 +73,10 @@ void ManagedSecondary::rawToMeta() {
 }
 
 bool ManagedSecondary::putMetadata(const Uptane::RawMetaPack &meta_pack) {
+  if (fiu_fail("secondary_putmetadata") != 0) {
+    return false;
+  }
+
```

Comment:
Can this go in VirtualSecondary instead of ManagedSecondary?

Reply:
Yes, though it needs an overriding of the method there but no big deal.